### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**
/CC @cvrebert

### CODE REVIEW

**Comment by XhmikosR** [comment]
So @twbs/bootlint, any thoughts?

**Comment by lirantal** [comment]
+1, packages are up to date.

**Comment by XhmikosR** [comment]
@cvrebert: ping

File: package.json [hunk 1]
```diff
@@ -38,17 +38,17 @@
...
+    "grunt-contrib-nodeunit": "^1.0.0",
+    "grunt-contrib-qunit": "^1.0.1",
     "grunt-contrib-watch": "^0.6.1",
-    "grunt-eslint": "^10.0.0",
+    "grunt-eslint": "^18.0.0",
     "grunt-exec": "^0.4.6",
-    "grunt-jscs": "^1.5.0",
-    "jquery": "~2.1.3",
+    "grunt-jscs": "^2.8.0",
+    "jquery": "~2.2.1",
```
Comment (XhmikosR) [inline package.json]: BTW, is the tilde intentional here?

Comment (cvrebert) [inline package.json]: Not really

**Comment by cvrebert** [comment]
Will try to look at this tomorrow.

File: package.json (hunk 1 above)
Comment (XhmikosR) [inline package.json]: OK I'm gonna change it then. Let me know if there are any issues with the pr

**Comment by cvrebert** [comment]
@XhmikosR Make that one alphabetization fix and then :shipit:

**Pull request merged**